### PR TITLE
Update 01_logging.md

### PR DESCRIPTION
**Pull request by ggerogiokas**
Adding some documentation so one knows how to point to handler names defined within a logging.yml configuration when using the python logger within kedro.

### CODE REVIEW

File: docs/source/08_logging/01_logging.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ log.warning("Issue warning")
 log.info("Send information")
 ```
 
+Note: Log names like the example above, `__name__`, can be handler names defined within the `handlers` section keys.  The default handler names are `console, info_file_handler, error_file_handler` found within `conf/base/logging.yml`.
```
Comment (AntonyMilneQB) [inline docs/source/08_logging/01_logging.md:21]:
````suggestion
```eval_rst
.. note:: The name of a logger corresponds to the keys of the ``loggers``  sections in ``logging.yml`` (e.g. ``kedro.io``). See `Python's logging documentation <https://docs.python.org/3/library/logging.html#logger-objects>`_ for more information.
```
````

**Review by AntonyMilneQB — COMMENTED** [review]
Hello and thank you for your contribution @ggerogiokas! What you wrote isn't quite correct - the argument to the function should be the name of a _logger_ rather than a _handler_. This isn't obvious - I had to try this out to check it.

I've suggested a change to correct this and also format the note a bit differently so it appears like this in our docs:
image

**Comment by ggerogiokas** [comment]
Looks great to me!  Yea, I was a bit confused with whether it was handler or logger name thanks for checking it out.  I guess all set to push after that.

**Review by AntonyMilneQB — APPROVED** [review]

**Comment by datajoely** [comment]
Thanks for doing this @ggerogiokas !

**Review by MerelTheisenQB — APPROVED** [review]
LGTM! 👍

**Pull request merged**